tests/unit/spec/base/mozilla-traffic-cop.js

```diff
@@ -40,6 +40,14 @@ describe('mozilla-traffic-cop.js', function () {
             cop.init();
             expect(cop.verifyConfig).toHaveBeenCalled();
         });
+
+        it('should initialize if DNT helper function is not available', function() {
+            var dntBak = window._dntEnabled;
+            window._dntEnabled = false;
```

Comment:
If we want to assert that the lib still initializes if the `dntEnabled` function is not available, would it be better to stub it and return `undefined` instead of setting it to `false`?